src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter2.tsx

```diff
@@ -33,13 +33,39 @@ const DEFAULT_CUSTOM_SIZE: CustomSize = {
   width: ["*", "*"],
 }
 
+const convertToCentimeters = (element: number) => Math.round(element * 2.54)
+
 const parseRange = (range?: string) => {
   return range?.split("-").map(s => {
     if (s === "*") return s
-    return parseInt(s, 10)
+    return convertToCentimeters(parseFloat(s))
   })
 }
 
+const convertRangeElementToInches = (element: number | "*") => {
+  if (element === "*") {
+    return element
+  }
+
+  return Math.round((element / 2.54) * 100) / 100
+}
+
+const convertRangeToInches = (range: CustomRange) => {
+  return [
+    convertRangeElementToInches(range[0]),
+    convertRangeElementToInches(range[1]),
+  ]
+}
+
+const convertSizeToInches = (size: CustomSize) => {
+  return {
+    height:
+      size.height !== undefined ? convertRangeToInches(size.height) : undefined,
+    width:
+      size.width !== undefined ? convertRangeToInches(size.width) : undefined,
```

Comment:
These sizes will be `undefined` if a user access an artwork grid from a URL with only one of the sizes (height or width) specified as a parameter.